pkg/deploy/deployment_che.go

```diff
@@ -83,6 +83,7 @@ func NewCheDeployment(cr *orgv1.CheCluster, cheImage string, cheTag string, cmRe
 
 	memLimit := util.GetValue(cr.Spec.Server.ServerMemoryLimit, DefaultServerMemoryLimit)
 	pullPolicy := corev1.PullPolicy(util.GetValue(string(cr.Spec.Server.CheImagePullPolicy), DefaultPullPolicyFromDockerImage(cheImageAndTag)))
+	cheDebug := util.GetValue(cr.Spec.Server.CheDebug, "false")
```

Comment:
Please consider moving "false" into some constants file, like we have with others above- - DefaultServerMemoryLimit, DefaultPullPolicyFromDockerImage....